acstools/polarization_tools.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python
+"""
+Toolkit for analyzing HST/ACS polarization data.
+
+CHANGE LOG
+----------
+2020-09-22: Creation. (T. Desjardins)
+"""
+
+import numpy as np
+import os
+import yaml
```

Comment:
I can't decide whether to make this package compulsory or optional dependency. Gotta think about it. Regardless, it needs to be defined in `setup.cfg`.